util-images/probes/pkg/dnspropagation/probe.go

```diff
@@ -153,12 +170,33 @@ func runSinglePod(client kubernetes.Interface, url string, podName string, names
 	klog.V(4).Infof("Starting dns propagation calculation for pod %s ...", url)
 	tick := time.NewTicker(interval)
 	defer tick.Stop()
+	var lookupErrorLogged = false
 	for {
 		select {
 		case <-tick.C:
 			klog.V(4).Infof("DNS lookup %s", url)
 			if err := lookup(url); err != nil {
-				klog.Warningf("DNS lookup error: %v", err)
+				if strings.Contains(err.Error(), "no such host") {
```

Comment:
nit: https://pkg.go.dev/net#DNSError DNSError struct contains isNotFound boolean flag that might be more reliable and convenient than string comparison